api/app.ts

```diff
@@ -18,7 +19,8 @@ dotenv.config();
 const connectionString = "mongodb://127.0.0.1:27017/calendar-app";
 const PORT = process.env.NODE_ENV === "test" ? 4001 : 4000;
 
-app.post("/entry", createCalendarEntry);
+app.post("/entries", createCalendarEntry);
```

Comment:
why is this plural entries? I thought we changed it to `entry` because we are creating a singular entry?

Reply:
Convention in restful routing is plural: https://www.learnhowtoprogram.com/c-and-net/basic-web-applications/introduction-to-restful-routing